frontend/src/app/components/crm/leads/leads.component.html

```diff
@@ -119,11 +122,17 @@
                  colspan="4">No data matching the filter "{{input.value}}"</mat-row>
       </mat-header-row>
     </mat-table>
-
+    <mat-card *ngIf="isLoading"
+              style="display: flex; justify-content: center; align-items: center">
+      <mat-progress-spinner
+        color="primary"
+        mode="indeterminate">
+      </mat-progress-spinner>
+    </mat-card>
     <!-- TODO: Switch this to use a class instead of setting styles -->
-    <mat-paginator [pageSizeOptions]="[5, 10, 20]"
+    <mat-paginator [pageSizeOptions]="[10, 5, 20, 30]"
```

Comment:
Can we make this something like 25, 50, 100?

Reply:
yep